server/jobs/job_store.go

```diff
@@ -150,9 +154,10 @@ func (s *StorageBackendJobStore) SetJobCompleteStatus(jobID string, fullRepoName
 
 	job, err := s.JobStore.Get(jobID)
 	if err != nil || job == nil {
-		return errors.Wrapf(err, "retrieveing job: %s from memory store", jobID)
+		return errors.Wrapf(err, "retrieving job: %s from memory store", jobID)
 	}
-
+	subScope := s.scope.SubScope("set_job_complete_status")
+	subScope.Counter("write_attempt").Inc(1)
```

Comment:
why attempt instead of success/error metrics?

Reply:
I wanted to compare attempt metrics to the actual success/failure metrics defined within the storage backend implementation